src/components/Dashboard/UserBox.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import classnames from 'classnames';
+import { useRouter } from 'next/router';
+
+import { useUserName } from '../AuthenticationProvider';
+import { useAuthentication } from '../AuthenticationProvider';
+import style from './UserBox.module.scss';
+
+export const UserBox = () => {
+  const [userBoxVisible, setUserBoxVisible] = useState(false);
+
+  const userName = useUserName();
+
+  const router = useRouter();
+
+  const { logout } = useAuthentication();
+
+  const handleRevealUserBox = () => {
+    setUserBoxVisible(!userBoxVisible);
+  };
+
+  const handleLogOut = () => {
+    logout();
+  };
+
+  const handleUpdate = () => {
+    router.push('/zmena');
+  };
+
+  return (
+    <div
+      className={classnames(style.userBox, {
+        [style.userBoxOpen]: userBoxVisible === true,
+      })}
+    >
+      <button onClick={handleRevealUserBox} className={style.profileBox}>
+        <div className={style.profile} />
+        <div
+          className={classnames(style.arrow, {
+            [style.arrowOpen]: userBoxVisible === true,
```

Comment:
You could make it shorter: 
```suggestion
            [style.arrowOpen]: userBoxVisible,
```

If you would somewhere need explicit boolean, do implicit casting with `!!userBoxVisible` that would return boolean

Reply:
changed on both cases